eZ/Bundle/EzPublishCoreBundle/Resources/config/services.yml

```diff
@@ -248,7 +248,7 @@ services:
         arguments:
             - '@ezpublish.api.repository'
         tags:
-            - { name: console.command, command: ezplatform:delete-content-translation }
+            - { name: console.command }
```

Comment:
This service is lazy, so we should technically be possible to avoid having to make this one lazy.

But most others will trigger load of repo indirectly, so maybe best to just let all be lazy so people use that and avoid hitting the issue.